src/lib/datepicker/datepicker.md

```diff
@@ -278,3 +317,41 @@ In year view:
 | `PAGE_DOWN`          | Go to next year                     |
 | `ALT` + `PAGE_DOWN`  | Go to next 10 years                 |
 | `ENTER`              | Select current month                |
+
+### Future work
+
+Currently the datepicker supports basic date selection functionality. There are more features that
+will be added in future iterations, including:
+ * Support for datetimes (e.g. May 2, 2017 at 12:30pm) and month + year only (e.g. May 2017)
+ * Support for selecting and displaying date ranges
+ * Support for custom time zones
+ * Infinite scrolling through calendar months
+ * Easier year selection
+ * Custom views for the calendar popup
```

Comment:
I would omit the future work section here; we don't include it in any of the other docs. Do we have issues to track these?